src/Test.Utilities/CodeMetricsTestsBase.cs

```diff
@@ -33,14 +35,18 @@ protected Project CreateProject(string[] sources, string language = LanguageName
 
             var projectId = ProjectId.CreateNewId(debugName: TestProjectName);
 
+            var defaultReferences = ReferenceAssemblies.NetFramework.Net48.Default;
+            var references = Task.Run(() => defaultReferences.ResolveAsync(LanguageNames.CSharp, CancellationToken.None)).GetAwaiter().GetResult();
```

Comment:
Can we avoid this test change in this PR?

Reply:
Well I need to change the references because of the test with the compiler generated attributes.

Reply:
See this test https://github.com/dotnet/roslyn-analyzers/pull/3315/files#diff-de51a4bb7ce3a178f017a5b53b974678R639

Reply:
Sounds good.